### PR TITLE
SAMBRO: format email

### PR DESCRIPTION
**Pull request by biplovbhandari**

### CODE REVIEW

File: modules/templates/SAMBRO/config.py [hunk 1]
```diff
@@ -1122,26 +1122,29 @@ def get_html_email_content(row, ack_id=None, system=True):
...
+                                                                      system=False,
+                                                                      ul=True)),
+                          })
                          if response_type else "",
-                         BR() if response_type else "",
-                         BR(),
                          BR() if instruction else "",
-                         T("Instructions: %(instruction)s") % \
-                         {"instruction": s3_str(get_formatted_value(instruction, system=system))}
+                         XML(T("<b>Instructions:</b> %(instruction)s") %
```
Comment (flavour) [inline modules/templates/SAMBRO/config.py]: Generally T() will work better for translators like this:
XML("<b>%(label)s:</b> %(instruction)s" %
{'label': T("Instructions"),

etc

File: modules/s3/s3resource.py [hunk 2]
```diff
@@ -6264,7 +6264,7 @@ def render(self,
         renderer = rfield.represent
         if not callable(renderer):
             # @ToDo: Don't convert unformatted numbers to strings
-            renderer = lambda v: s3_unicode(v) if v is not None else none
+            renderer = lambda v: s3_unicode(v) if v is not None else None
```
Comment (nursix) [inline modules/s3/s3resource.py]: Nope - indeed none (parameter of this function), and not None.

**Comment by biplovbhandari** [comment]
@flavour Please merge

File: modules/templates/SAMBRO/config.py [hunk 3]
```diff
@@ -1085,13 +1085,17 @@ def get_html_email_content(row, ack_id=None, system=True):
...
+                         {"label": T("<b>Note</b>"),
+                          "note": s3_str(note),
+                          })
                          if note else "",
                          BR() if note else "",
                          BR() if note else "",
-                         T("ID: %(identifier)s") % \
-                         {"identifier": s3_str(row["cap_alert.identifier"])},
+                         XML("%(label)s: %(identifier)s" %
+                         {"label": T("<b>ID</b>"),
```
Comment (flavour) [inline modules/templates/SAMBRO/config.py]: Keep the HTML formatting outside of T:
XML("<b>%(label)s</b>:
{"label": T("ID"),

**Comment by biplovbhandari** [comment]
@flavour Please merge

**Pull request merged**